modules/howtos/examples/query-consistency.cc

```diff
@@ -0,0 +1,154 @@
+#include <string>
```

Comment:
is this file a duplicate of `modules/devguide/examples/c/query-consistency.cc`? if so, could you use symlink instead to avoid duplication?